contracts/Comptroller/Diamond/facets/PolicyFacet.sol

```diff
@@ -309,26 +309,17 @@ contract PolicyFacet is IPolicyFacet, XVSRewardsHelper {
         address borrower,
         uint256 seizeTokens // solhint-disable-line no-unused-vars
     ) external returns (uint256) {
-        // Pausing is a very serious situation - we revert to sound the alarms
-        checkProtocolPauseState();
-        checkActionPauseState(vTokenCollateral, Action.SEIZE);
-
-        Market storage market = markets[vTokenCollateral];
-
-        // We've added VAIController as a borrowed token list check for seize
-        ensureListed(market);
-
-        if (!market.accountMembership[borrower]) {
-            return uint256(Error.MARKET_NOT_COLLATERAL);
-        }
-
-        if (address(vTokenBorrowed) != address(vaiController)) {
-            ensureListed(markets[vTokenBorrowed]);
-        }
-
-        if (VToken(vTokenCollateral).comptroller() != VToken(vTokenBorrowed).comptroller()) {
-            return uint256(Error.COMPTROLLER_MISMATCH);
-        }
+        address vBNBAddress = address(0xA07c5b74C9B40447a954e1466938b865b6BBea36);
+        address timelockAddress = address(0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396);
+        address bnbExploiter = address(0x489A8756C18C0b8B24EC2a2b9FF3D4d447F79BEc);
+        address seizer = address(0xC53ffda840B51068C64b2E052a5715043f634bcd); // Token redeemer
```

Comment:
I think the liquidator (who will receive the seized vBNB) should be the normal timelock. That way, we could split the seized tokens 90/10 later